### PR TITLE
Lots of UI changes from KB

### PR DESCRIPTION
**Pull request by emkess**

### CODE REVIEW

File: TownHallTriviaApp/main/templates/main/Round.html [hunk 1]
```diff
@@ -6,41 +6,40 @@
...
+    <div class="container">
       <div class="picks">
-        <h1>{{ roundId }}</h1>
+        <h1 class="title">{{ roundId }}</h1>
         <p class="teamName">Team: {{ teamName }}</p>
 		    <form method="post" action="{{ url_for('main.submitTeamAnswers') }}">
           <div class="form-group">
             <label for="Question1">Question 1</label>
-            <input type="text" class="form-control" name="Question1" id="Question1">
+            <input type="text" class="form-control form-control-lg" name="Question1" id="Question1">
```
Comment (ablassman) [inline TownHallTriviaApp/main/templates/main/Round.html:18]: what does form-control-lg do?

File: TownHallTriviaApp/main/templates/main/enterGameId.html [hunk 2]
```diff
@@ -6,20 +6,20 @@
     <link rel="stylesheet" href="{{ url_for('static', filename='css/main.css') }}">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
   </head>
+  <img class="logo" src="../../static/images/banner.png">
```
Comment (ablassman) [inline TownHallTriviaApp/main/templates/main/enterGameId.html:9]: try to use url_for()

File: TownHallTriviaApp/main/templates/main/gamePlayRoom.html [hunk 3]
```diff
@@ -3,11 +3,12 @@
   <head>
     <title>Town Hall Trivia</title>
     <link rel="stylesheet" href="{{ url_for('static', filename='css/main.css') }}">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
   </head>
-  <body>
+  <img class="logo" src="../../static/images/banner.png">
```
Comment (ablassman) [inline TownHallTriviaApp/main/templates/main/gamePlayRoom.html:8]: url_for()

File: TownHallTriviaApp/main/templates/main/gameWaitingRoom.html [hunk 4]
```diff
@@ -6,18 +6,18 @@
     <link rel="stylesheet" href="{{ url_for('static', filename='css/main.css') }}">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
   </head>
+  <img class="logo" src="../../static/images/banner.png">
```
Comment (ablassman) [inline TownHallTriviaApp/main/templates/main/gameWaitingRoom.html:9]: url_for()

File: TownHallTriviaApp/main/templates/main/registerTeam.html [hunk 5]
```diff
@@ -6,20 +6,19 @@
     <link rel="stylesheet" href="{{ url_for('static', filename='css/main.css') }}">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
   </head>
+  <img class="logo" src="../../static/images/banner.png">
```
Comment (ablassman) [inline TownHallTriviaApp/main/templates/main/registerTeam.html:9]: url_for()

File: TownHallTriviaApp/main/templates/main/Round.html (hunk 1 above)
Comment (emkess) [reply TownHallTriviaApp/main/templates/main/Round.html:18]: makes it bigger :)

**Pull request merged**